### PR TITLE
Add precompile bridge(transferNative) test case

### PR DESCRIPTION
**Pull request by 0xverin**
### Context

resolves p-362

### CODE REVIEW

**Comment by 0xverin** [comment]
I want to run a CI test to see if it can be executed successfully. It hasn't reached the review state yet cc @Traf333

**Comment by 0xverin** [comment]
> ~~I want to run a CI test to see if it can be executed successfully. It hasn't reached the review state yet cc @Traf333~~

ready for review.

File: ts-tests/integration-tests/precompile-contract.test.ts [hunk 1]
```diff
@@ -229,28 +265,71 @@ describeLitentry('Test Parachain Precompile Contract', ``, (context) => {
...
+            '0xaaafB3972B05630fCceE866eC69CdADd9baC2772', // random address
+            0
+        );
+
+        const res = await executeTransaction(transferNativeTx, precompileBridgeContractAddress, 'transferNative');
+        expect(res.status).to.eq(true);
+
+        const eventsPromise = subscribeToEvents('chainBridge', 'FungibleTransfer', context.api);
+        const events = (await eventsPromise).map(({ event }) => event);
+        expect(events.length).to.eq(1);
```
Comment (wangminqi) [inline ts-tests/integration-tests/precompile-contract.test.ts:334]: It might be better to compare the event content though

**Review by wangminqi — CHANGES_REQUESTED** [review]

**Comment by wangminqi** [comment]
overall, LGTM

File: ts-tests/integration-tests/precompile-contract.test.ts (hunk 1 above)
Comment (0xverin) [reply ts-tests/integration-tests/precompile-contract.test.ts:334]: Added checks for event data.

**Review by wangminqi — APPROVED** [review]

**Pull request merged**